### ISSUE TITLE
Invalid node type when using dadbod-ui

### ISSUE THREAD

**Issue by sho-87** [DerekStride/tree-sitter-sql#142]
I cant seem to get treesitter highlighting working when writing queries inside [vim-dadbod-ui](https://github.com/kristijanhusak/vim-dadbod-ui)

I keep getting the following error:

![image](https://user-images.githubusercontent.com/5199715/235414366-3fc15eb3-2f52-43da-a8d1-c173eb885f66.png)

**Comment by matthias-Q** [comment]
I have checked it and I do not have this issue. Can you give us a bit more information? From the screenshot it is quite hard to figure out the issue.

When you are in the query buffer, then run `:InspectTree` and paste the result (maybe add the query as well if possible). This will work if you are running neovim >=0.9.0

**Comment by sho-87** [comment]
a little bit more context here:
![image](https://github.com/DerekStride/tree-sitter-sql/assets/5199715/803091f7-9b95-472f-9652-6ed19e7377ad)

basically, I have a sqlite db that has a few tables in it. I try a simple `select * from <table>` and I get a ts highlighter error as shown in image.

I have also included the tree in the image, but not familiar enough with ts to understand it. Although, I do notice that when I hover the cursor on the `(keyword_limit)` line, that is when the 2nd highlighter shows up

This doesn't happen with all sqlite files. I tried with a different file last night and the highlighting seemed correct, so I'm not sure what differences might be causing errors in one case and not in the other
